src/react/components/action-bar/action-bar.tsx

```diff
@@ -38,11 +38,13 @@ export class SurveyActionBar extends SurveyElementBase<IActionBarProps, any> {
     super.componentDidMount();
     if (!this.model.hasActions) return;
     const container: HTMLDivElement = this.rootRef.current;
-    this.manager = new ResponsivityManager(
-      container,
-      (this.model as any),
-      ".sv-action:not(.sv-dots)>.sv-action__content"
-    );
+    if (this.model.createResponsivityManager) {
+      this.manager = new ResponsivityManager(
+        container,
+        (this.model as any),
+        ".sv-action:not(.sv-dots)>.sv-action__content"
+      );
+    }
   }
   componentWillUnmount() {
     this.manager && this.manager.dispose();
```

Comment:
What about VueJS?

Reply:
VueJS action bar does not support ResponsivityManager at this moment. We should discuss the priority of its implementation.